### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

File: raddb/mods-available/always [hunk 1]
```diff
@@ -10,8 +10,6 @@
 #
 #  The `always` module simply returns the same result, always, without
 #  doing anything else.  It is here for use in complex policies.
-#
-
```
Comment (alandekok) [inline raddb/mods-available/always]: I really don't like deleting these trailing comments.  The example configs use this a *lot*

File: raddb/mods-available/dhcpv4 [hunk 2]
```diff
@@ -28,7 +28,7 @@
 #
 
 #
-#  # Default instance
+#  ## Default instance
```
Comment (alandekok) [inline raddb/mods-available/dhcpv4]: We should decide if it's `==` or `##`, and use one version everywhere.

File: raddb/mods-available/rest [hunk 3]
```diff
@@ -3,6 +3,8 @@
 #  :toc:
 #
 #  $Id$
+#
+#
```
Comment (alandekok) [inline raddb/mods-available/rest]: Why two blank comments here?

Comment (jpereira) [reply raddb/mods-available/rest]: it was a typo.

File: raddb/clients.conf [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #  -*- text -*-
 #
-#  :toc:
+#  :toc: left
```
Comment (alandekok) [inline raddb/clients.conf]: If it's possible, this should go into either a style sheet, or maybe a command-line option.

File: doc/all.mk [hunk 5]
```diff
@@ -102,7 +102,7 @@ doc/raddb/%.adoc: raddb/%
 
 doc/%.html: doc/%.adoc
 	@echo HTML $^
-	${Q}$(ASCIIDOCTOR) $< -b html5 -o $@ $<
+	${Q}$(ASCIIDOCTOR) $< -b html5 -T ./scripts/asciidoc/freeradius.template -o $@ $<
```
Comment (alandekok) [inline doc/all.mk]: This requires the gem `tilt`:

```
asciidoctor: FAILED: 'tilt' could not be loaded
You must have the tilt gem installed (gem install tilt) to use custom backend templates
```

Seems like a good idea, tho.  I would be happier if asciidoc would ignore the argument if the gem wasn't present.

File: raddb/clients.conf (hunk 4 above)
Comment (alandekok) [reply raddb/clients.conf]: `asciidoc-a toc=left FILE`

Maybe we just want to set this as the default for all of the files.

File: doc/all.mk (hunk 5 above)
Comment (jpereira) [reply doc/all.mk]: the `tilt` could be checked as we do for `asciidoctor` and `pandoc`?

File: raddb/clients.conf (hunk 4 above)
Comment (jpereira) [reply raddb/clients.conf]: it was a typo. :(

File: raddb/mods-available/dhcpv4 (hunk 2 above)
Comment (jpereira) [reply raddb/mods-available/dhcpv4]: yes, I noticed that the `#` is a _markdown_ stuff. I will update to use the `=` levels.

File: doc/all.mk (hunk 5 above)
Comment (alandekok) [reply doc/all.mk]: Likely best to check at the `configure` stage.

But checking the template, it should really be applied at the PDF stage.   The file `scripts/asciidoc/freeradius.template` is LaTex, and not HTML.

We will likely need a separate HTML / CSS template for the HTML output.  Right now, the colors are red and black, which doesn't match the default FreeRADIUS look.

File: raddb/mods-available/dhcpv4 (hunk 2 above)
Comment (alandekok) [reply raddb/mods-available/dhcpv4]: There are still a lot of `##` instead of `==`

Comment (alandekok) [reply raddb/mods-available/dhcpv4]: Maybe update the `==` as a separate commit?

**Pull request merged**